src/matrixLabels/vectorizeLabel.js

```diff
@@ -1,14 +1,14 @@
 import vectorize_text from "vectorize-text";
 
-export default function vectorize_label(fontDetails, inst_axis, inst_name, async=true) {
+export default function vectorize_label(fontDetails, inst_axis, inst_name) {
   const vect_text_attrs = {
     textAlign: "left",
     triangles: true,
     size: fontDetails,
     font: '"Open Sans", verdana, arial, sans-serif',
   };
 
-  if (async) {
+  if (OffscreenCanvas) {
```

Comment:
This will fail under webworker with env that does not support OffscreenCanvas for instance Firefox.
Besides, if OffscreenCanvas is not supported, we could not fire vectorize_text in webworker since it will require a canvas.
For the fallback, I think it will make more sense if we do it without using worker.

Reply:
Can we just change it to `typeof OffscreenCanvas === "undefined"`? I didn’t realize there was a difference if it didn’t exist at all, sorry